pkgs/top-level/all-packages.nix

```diff
@@ -42054,4 +42054,6 @@ with pkgs;
   insulator2 = callPackage ../applications/misc/insulator2 {};
 
   animdl = python3Packages.callPackage ../applications/video/animdl { };
+
+  multiqc = callPackage ../applications/science/biology/multiqc { };
```

Comment:
I'd use `pkgs/by-name` and avoid editing the way too large `all-packages.nix`.

Reply:
Nowadays by-name is mandatory